doxy-my-docs.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+
+import doxygen
+import logging
+import os
+
+import sys
+
+from doxymydocs import configurationEnum, AppConfiguration
+from typing import Optional
+
+
+def __update_doxygen_config() -> str:
+    """
+    Update the doxygen configuration from the configuration
+
+    :return: The path to the updated doxygen configuration
+    """
+
+    logging.info("update Doxygen configuration")
+
+    doxyfile = AppConfiguration().get_doxygen_config()[configurationEnum.Doxygen.DOXYFILE]
+    logging.debug("Doxyfile: {}".format(doxyfile))
+
+    project_config = AppConfiguration().get_project_config()
+
+    doxygen_config_parser = doxygen.ConfigParser()
+    doxygen_configuration = doxygen_config_parser.load_configuration(doxyfile)
+
+    if configurationEnum.Project.VERSION in project_config:
+        logging.debug("Update PROJECT_NUMBER from {} to {}".format(doxygen_configuration['PROJECT_NUMBER'], project_config[configurationEnum.Project.VERSION]))
```

Comment:
Why use `configurationEnum.Project.VERSION` and not `configurationEnum.Project.NUMBER` ?

Reply:
Host My Docs setting name

Reply:
Normalized with HostMyDocs config and  for me `Project.NUMBER` is lesser clear than `Project.
VERSION`